x-pack/plugins/security_solution/public/management/pages/policy/view/protection_updates/protection_updates_layout.tsx

```diff
@@ -0,0 +1,347 @@
+            color={'primary'}
+            iconType={'iInCircle'}
+            size="m"
+            data-test-subj="protection-updates-automatic-updates-enabled"
+            title={i18n.translate(
+              'xpack.securitySolution.endpoint.protectionUpdates.automaticUpdates.enabledTitle',
+              {
+                defaultMessage: 'Automatic updates enabled',
+              }
+            )}
+          >
+            {i18n.translate(
+              'xpack.securitySolution.endpoint.protectionUpdates.automaticUpdates.enabled',
+              {
+                defaultMessage:
+                  'Manifest will always be updated to the latest available version. If you want to control updates manually, disable "Update manifest automatically".',
+              }
+            )}
+          </EuiCallOut>
+        );
+      }
+      return (
+        <>
+          <EuiTitle
+            size="xxs"
+            data-test-subj={'protection-updates-manifest-name-deployed-version-title'}
+          >
+            <h5>
+              {i18n.translate(
+                'xpack.securitySolution.endpoint.protectionUpdates.deployedVersion.label',
+                {
+                  defaultMessage: 'Currently deployed version',
+                }
+              )}
+            </h5>
+          </EuiTitle>
+          <EuiSpacer size="m" />
+
+          <EuiText size="m" data-test-subj="protection-updates-deployed-version">
+            {deployedVersion === 'latest' ? 'latest' : formattedDate}
+          </EuiText>
+          <EuiSpacer size="l" />
+          {renderVersionToDeployPicker()}
+
+          <EuiSpacer size="m" />
+
+          <EuiButton
+            fill={true}
+            disabled={!canWritePolicyManagement}
+            iconType="save"
+            data-test-subj="policyDetailsSaveButton"
+            onClick={onSave}
+            isLoading={isUpdating}
+          >
+            <FormattedMessage
+              id="xpack.securitySolution.endpoint.protectionUpdates.saveButton"
+              defaultMessage="Save updates"
+            />
+          </EuiButton>
+        </>
+      );
+    };
+
+    return (
+      <EuiPanel
+        data-test-subject="protection-updates-layout"
+        hasBorder={true}
+        hasShadow={false}
+        paddingSize="none"
+      >
+        <EuiFlexGroup
+          direction="row"
+          gutterSize="none"
+          alignItems="center"
+          style={{ padding: `${paddingSize} ${paddingSize} 0 ${paddingSize}` }}
+        >
+          <EuiFlexItem grow={1}>
+            <EuiTitle size="xxs" data-test-subj={'protection-updates-manifest-name-title'}>
+              <h5>
+                {i18n.translate('xpack.securitySolution.endpoint.protectionUpdates.manifestName', {
+                  defaultMessage: 'Manifest name',
+                })}
+              </h5>
+            </EuiTitle>
+            <EuiText size="m" data-test-subj="protection-updates-manifest-name">
+              {'artifactsec'}
```

Comment:
this looks like unintended

Reply:
It is intended, at least for now ;) 
https://github.com/elastic/security-team/issues/7208#issuecomment-1686627929

Reply:
oh, I see, thanks!